lib/ui/onboard/home_page.dart

```diff
@@ -1,4 +1,4 @@
-// ignore_for_file: use_build_context_synchronously, deprecated_member_use
+// ignore_for_file: deprecated_member_use
```

Comment:
remove flutter lint ignore rules

Reply:
Can't remove due to a dependency being used which is required.